tecken/base/symboldownloader.py

```diff
@@ -48,14 +50,40 @@ def wrapper(self, *args, **kwargs):
     miss_callable=lambda *a, **k: metrics.incr("symboldownloader_exists_cache_miss", 1),
 )
 @metrics.timer_decorator("symboldownloader_exists")
-def check_url_head(url):
+def get_last_modified(url: str) -> datetime.datetime:
```

Comment:
Huh, I meant to write `datetime.datetime | None`, but somehow didn't. I personally prefer using `| None` over `Optional`, since I think its meaning is more obvious, it doesn't require an additional import and it's shorter. (I think the syntax was introduced in Python 3.10.)

Reply:
Let's use `Optional` everywhere then.